tools/dockerfile/test/python_alpine_x64/Dockerfile

```diff
@@ -32,14 +34,21 @@ RUN apk update && apk add \
   strace \
   python-dev \
   py-pip \
+  python3-dev \
   unzip \
   wget \
   zip
 
-# Install Python packages from PyPI
-RUN pip install --upgrade pip==10.0.1
-RUN pip install virtualenv
-RUN pip install futures==2.2.0 enum34==1.0.4 protobuf==3.5.0.post1 six==1.10.0
+# Install Python 2 for tooling code.
+RUN pip install --upgrade pip==10.0.1 && \
+	pip install virtualenv futures==2.2.0 enum34==1.0.4 \
+		protobuf==3.5.0.post1 six==1.10.0
+
+# Install Python 3 packages from PyPI
+RUN python3 -m ensurepip && \
+	python3 -m pip install --upgrade pip==10.0.1 && \
```

Comment:
Should we use newer `pip` here?

Reply:
I don't think it would be a bad idea to just do `pip install --upgrade pip` with no version specifier. But I was trying to follow the patterns set by the existing Dockerfiles. You could argue that there's value in knowing that you work with old versions of Pip.

Reply:
Done.